packages/story-editor/src/components/panels/document/taxonomies/test/taxonomies.js

```diff
@@ -25,9 +25,20 @@ import { screen } from '@testing-library/react';
 import TaxonomyContext from '../../../../../app/taxonomy/context';
 import { renderWithTheme } from '../../../../../testUtils';
 import TaxonomiesPanel from '../taxonomies';
+import { StoryContext } from '../../../../../app/story';
 
 function arrange({ taxonomies }) {
   const storyContextValue = {
+    state: {
+      capabilities: taxonomies.reduce((acc, curr) => {
+        acc[`assign-${curr.slug}`] = true;
+        acc[`create-${curr.slug}`] = true;
+        return acc;
+      }, {}),
```

Comment:
@swissspidy  thank you for all of this, i see where I veered off course.

Reply:
happy to help!